### PR TITLE
Reviewlist- clean up codes and implement helpful button

### PR DESCRIPTION
**Pull request by cdeng1111**
@fernanda1411 @van-hsieh can you review?

### CODE REVIEW

File: client/src/components/RatingsAndReviews/WritingReview/WritingReview.jsx [hunk 1]
```diff
@@ -28,150 +28,152 @@ class WritingReview extends React.Component {
...
-        <div>
+      <div>
           <p>your nickname:
             <input name='name' type='text' value={name} onChange={this.handleChange} />
           </p>
-          </div>
-          <div>
+      </div>
+      <div>
             <p>your email:
```
Comment (fernanda1411) [inline client/src/components/RatingsAndReviews/WritingReview/WritingReview.jsx]: If you want to do the email validation. This is how I did on mine 
https://github.com/sensational-sycamore/sycamore/blob/27982cf2c10f045bf74a075abd38202ff10db1cc/client/src/components/QuestionsAndAnswers/AddAnswerModal/AddAnswerModal.jsx#L88

**Review by fernanda1411 — APPROVED** [review]
Good Work!!! I left a comment.

**Review by van-hsieh — COMMENTED** [review]
Looks good to me!

**Review by fernanda1411 — APPROVED** [review]
looks good to me :)

File: client/src/components/RatingsAndReviews/WritingReview/WritingReview.jsx (hunk 1 above)
Comment (cdeng1111) [reply client/src/components/RatingsAndReviews/WritingReview/WritingReview.jsx]: added email validation

**Pull request merged**